programs/sbf/Cargo.toml

```diff
@@ -25,30 +25,30 @@ rand = "0.8"
-solana-program-runtime = { path = "../../program-runtime", version = "=1.19.0" }
-solana-program-test = { path = "../../program-test", version = "=1.19.0" }
-solana-runtime = { path = "../../runtime", version = "=1.19.0" }
-solana-sbf-rust-128bit-dep = { path = "rust/128bit_dep", version = "=1.19.0" }
-solana-sbf-rust-invoke = { path = "rust/invoke", version = "=1.19.0" }
-solana-sbf-rust-invoked = { path = "rust/invoked", version = "=1.19.0", default-features = false }
-solana-sbf-rust-many-args-dep = { path = "rust/many_args_dep", version = "=1.19.0" }
-solana-sbf-rust-mem = { path = "rust/mem", version = "=1.19.0" }
-solana-sbf-rust-param-passing-dep = { path = "rust/param_passing_dep", version = "=1.19.0" }
-solana-sbf-rust-realloc = { path = "rust/realloc", version = "=1.19.0", default-features = false }
-solana-sbf-rust-realloc-invoke = { path = "rust/realloc_invoke", version = "=1.19.0" }
-solana-sdk = { path = "../../sdk", version = "=1.19.0" }
-solana-transaction-status = { path = "../../transaction-status", version = "=1.19.0" }
-agave-validator = { path = "../../validator", version = "=1.19.0" }
-solana-zk-token-sdk = { path = "../../zk-token-sdk", version = "=1.19.0" }
-solana-svm = { path = "../../svm", version = "=1.19.0" }
+solana-account-decoder = { path = "../../account-decoder", version = "=2.0.0" }
+solana-accounts-db = { path = "../../accounts-db", version = "=2.0.0" }
+solana-bpf-loader-program = { path = "../bpf_loader", version = "=2.0.0" }
+solana-cli-output = { path = "../../cli-output", version = "=2.0.0" }
+solana-ledger = { path = "../../ledger", version = "=2.0.0" }
+solana-logger = { path = "../../logger", version = "=2.0.0" }
+solana-measure = { path = "../../measure", version = "=2.0.0" }
+solana-program = { path = "../../sdk/program", version = "=2.0.0" }
+solana-program-runtime = { path = "../../program-runtime", version = "=2.0.0" }
+solana-program-test = { path = "../../program-test", version = "=2.0.0" }
+solana-runtime = { path = "../../runtime", version = "=2.0.0" }
+solana-sbf-rust-128bit-dep = { path = "rust/128bit_dep", version = "=2.0.0" }
+solana-sbf-rust-invoke = { path = "rust/invoke", version = "=2.0.0" }
+solana-sbf-rust-invoked = { path = "rust/invoked", version = "=2.0.0", default-features = false }
+solana-sbf-rust-many-args-dep = { path = "rust/many_args_dep", version = "=2.0.0" }
+solana-sbf-rust-mem = { path = "rust/mem", version = "=2.0.0" }
+solana-sbf-rust-param-passing-dep = { path = "rust/param_passing_dep", version = "=2.0.0" }
+solana-sbf-rust-realloc = { path = "rust/realloc", version = "=2.0.0", default-features = false }
+solana-sbf-rust-realloc-invoke = { path = "rust/realloc_invoke", version = "=2.0.0" }
+solana-sdk = { path = "../../sdk", version = "=2.0.0" }
+solana-svm = { path = "../../svm", version = "=2.0.0" }
+solana-transaction-status = { path = "../../transaction-status", version = "=2.0.0" }
+agave-validator = { path = "../../validator", version = "=2.0.0" }
```

Comment:
nittiest-possible-nit: this breaks alphabetical ordering, but it might make more sense to keep it in the same cluster as all the monorepo crates

Reply:
I'm fine with either ordering, but if we decide to change it let's do it in another PR. Syncing commits back to the Labs monorepo is much simpler that way.